homeassistant/components/openexchangerates/sensor.py

```diff
@@ -48,20 +48,6 @@ async def async_setup_platform(
         severity=IssueSeverity.WARNING,
         translation_key="deprecated_yaml",
     )
```

Comment:
Please remove the issue and platform setup and platform schema too.

Reply:
I looked at some others where it seems the issue creation was left behind so it would still report an issue even if no import was so done which I think could make sense. 
So this should not be the case then and a creation of an issue should also be removed?

Reply:
I'd rather just clean this up. I think we can decide on a case per case basis if we need an issue. In this case I don't think we need to keep it.